### PR TITLE
Jamie/deploy script with solver

### PR DESCRIPTION
**Pull request by jpick713**

### CODE REVIEW

File: script/deploy/07_DeployRolesAuthority.s.sol
Comment (CarsonCase) [inline script/deploy/07_DeployRolesAuthority.s.sol:1]: What do we think about moving the ROLE numbers to a constants file somewhere?

**Review by CarsonCase — COMMENTED** [review]
Looks good! Only thought is if we should move ROLEs to a constants file to keep them consistent

File: script/deploy/07_DeployRolesAuthority.s.sol
Comment (jpick713) [reply script/deploy/07_DeployRolesAuthority.s.sol:1]: good idea, will refactor now

**Comment by jpick713** [comment]
refactored into other repo

**Pull request closed without merging**